airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicPerformanceTest.kt

```diff
@@ -134,16 +134,19 @@ abstract class BasicPerformanceTest(
 
     @Test
     open fun testInsertRecords() {
-        testInsertRecords(null)
+        testInsertRecords(validation = null)
     }
 
-    protected fun testInsertRecords(validation: ValidationFunction?) {
+    protected fun testInsertRecords(
+        recordsToInsert: Long? = null,
+        validation: ValidationFunction?,
```

Comment:
Nit...why not just default this to `null` here to avoid having to pass it to the method to begin with?

Reply:
A few things at play in here:
* There's an `@Test` annotated parent method, from junit and our style checks, cannot return a value, parameters cannot take default
* The inner method was introduced so that connectors could override their own validation
* If we make validation optional, it runs into ambiguous method calls since both can take no args, but because of our checks, we cannot just have one method.